javascript-data-model/order-history.js

```diff
@@ -0,0 +1,55 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    bookName: 'JavaScript for Impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    bookImage: 'images/images1.png',
+    ordered: 'August 4, 2020',
+    total: '$34.00',
+    bookRetailPrice: '$31.55',
+    shipTo: 'JS Masher',
+    delivered: 'Delivered Aug 8, 2020',
+    deliveryLocation: 'Your package was delivered.It was handed directly to a resident.',
+    returnBy: 'Return window Closed on Sep 7, 2020'
```

Comment:
This still has all the data for the item(s) in the same object as the data about the order itself. You should make a property on the order object to store all the data for all the items in the order.